pkgs/top-level/all-packages.nix

```diff
@@ -39654,6 +39654,10 @@ with pkgs;
 
   gedit = callPackage ../applications/editors/gedit { };
 
+  gerbolyze = callPackage ../by-name/ge/gerbolyze/package.nix {
+    pythonPackages = python3Packages;
```

Comment:
Perhaps in gerbolyze's package.nix you could pass `pythonPackages` as a function argument that defaults to `python3Packages`? That would let you remove this entry.

Reply:
I was considering this, but didn’t go that route - I think this change would be fine.

Reply:
Changed to an explicit dependency on `python3Packages`.